### PR TITLE
test: adding tests for setting up governance params for liquidation flow and ci changes

### PR DESCRIPTION
**Pull request by rabi-siddique**
The PR does the following:
- Add tests for setting up manager and auctioneer params for Liquidation. 
- Update CI configurations to execute these tests.

This is the first PR for adding tests for implementing Happy Path Scanerio Liquidation. It will be followed by subsequent PRs containing other tests.

### CODE REVIEW

File: test/support.js [hunk 1]
```diff
@@ -1 +1 @@
-import '@agoric/synpress/support/index';
```
Comment (frazarshad) [inline test/support.js:1]: reason for changing this? `index should work just fine`

File: test/e2e/specs/test.spec.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 /* eslint-disable ui-testing/no-disabled-tests */
 describe('Wallet App Test Cases', () => {
   context('Test commands', () => {
-    it(`should connect with Agoric Chain`, () => {
+    it(`should setup wallet and connect with Agoric Chain`, () => {
+      cy.setupWallet();
```
Comment (frazarshad) [inline test/e2e/specs/test.spec.js:5]: we dont have the `SKIP_EXTENSION_SETUP` flag enabled anywhere, wont this duplicate the setup of the wallet?

File: test/support.js (hunk 1 above)
Comment (rabi-siddique) [reply test/support.js:1]: I have customized the `support.js` file in `@agoric/synpress`. In the `before` hook, we originally set up the wallet.  That sets the wallet every time I visit a different origin. I found this approach to be excessive, so I opted to override it. Now, I configure the wallet setup explicitly within a specific test to ensure we don't set the wallet everytime we visit a different origin.

Comment (rabi-siddique) [reply test/support.js:1]: I was wondering if we should remove setting up wallet in the `before` hook?

**Pull request closed without merging**